Add Delvan Shipworm missions for Delve and Ada (#12376)
endless-sky/endless-sky: data/human/culture conversations.txt

			`	MISSING:`
			`		Anna Fangvin, a young author whose books have been shared throughout the planet. Please report any news to the Coalition of Book Enjoyers, address 5662 Myrtle Lane, Sizertown, Zug.`
				decline

mission "Exotic Life: Delve"
	minor
	source Delve
	to offer
		random < 20
	on offer

> 		log `Heard about some tiny creatures called Delvan Shipworms which can eat through a ship's hull. Apparently infestations are getting more frequent.`

Author: ```suggestion
		log `Heard about some tiny creatures called Delvan shipworms which can eat through a ship's hull. Apparently infestations are getting more frequent.`
```

> 		log `Heard about some tiny creatures called "Delvan Shipworms" which can eat through a ship's hull. Apparently infestations are getting more frequent.`

Reviewer: Hmm... not sure you need the quotes around `Delvan Shipworms`. It could go either way.

> 		log `Heard about some tiny creatures called "Delvan Shipworms" which can eat through a ship's hull. Apparently infestations are getting more frequent.`

Author: ```suggestion
		log `Heard about some tiny creatures called Delvan Shipworms which can eat through a ship's hull. Apparently infestations are getting more frequent.`
```

		conversation
			`The mountain air tastes cool and fresh after breathing the constantly recycled air on the <ship> for so long, so you take a moment to get a good lungful and enjoy the view. Just off to your right, ground technicians wearing thin coveralls and breathing gear are spraying some kind of chemical on the hull of a parked Sunder. The fine mist reflects a bright rainbow which contrasts strikingly with the dull gray metal of the hull.`

> 			`	You wander over, careful not to get too close to the spray, which you assume is toxic. One of the technicians climbs down a ladder attached to the hull as you approach, then turns to you, pulling their mask off to reveal a man in his early twenties, with dark skin and hair plastered with sweat to his forehead.`

Reviewer: "plastered with sweat" feels a bit awkward. Maybe something like this would flow better?

```suggestion
			`	You wander over, careful not to get too close to the spray, which you assume is toxic. One of the technicians climbs down a ladder attached to the hull as you approach, then turns to you, pulling their mask off to reveal a dark-skinned man in his early twenties, his hair plastered to his forehead by copious amounts of sweat.`
```

			`	"This your ship?" he asks with a professionally neutral expression.`
			choice
				`	"No, I'm just curious. What are you doing to it?"`
					goto curious
				`	"Yes. How is the work coming along?"`
			`	The technician shakes his head morosely.`

> 			`	"It's confirmed. Shipworm. Bad case too, I'm surprised your hull was still holding atmo." He looks back at the Sunder, measuring her up with his eyes. "We should be finished with the decontamination in a couple hours, but then we'll need to go over her with sealant. Probably two coats. We'll be finished tomorrow afternoon, earliest."`

Reviewer: It works as-is since it's dialogue, but this would be more grammatically correct.

```suggestion
			`	"It's confirmed. Shipworm. Bad case, too - I'm surprised your hull was still holding atmo." He looks back at the Sunder, measuring it up with his eyes. "We should be finished with the decontamination in a couple hours, but then we'll need to go over her with sealant. Probably two coats. We'll be finished tomorrow afternoon, earliest."`
```

> 			`	"It's confirmed. Shipworm. Bad case, too - I'm surprised your hull was still holding atmo." He looks back at the Sunder, measuring it up with his eyes. "We should be finished with the decontamination in a couple hours, but then we'll need to go over her with sealant. Probably two coats. We'll be finished tomorrow afternoon, earliest."`

Reviewer 2: Could be worth expanding atmo for the benefit of players who aren't native English speakers.

```suggestion
			`	"It's confirmed. Shipworm. Bad case, too - I'm surprised your hull was still holding atmosphere." He looks back at the Sunder, measuring it up with his eyes. "We should be finished with the decontamination in a couple hours, but then we'll need to go over her with sealant. Probably two coats. We'll be finished tomorrow afternoon, earliest."`
```

			choice

> 				`	"Shipworm? How could this happen?"`

Reviewer 2: "How could this happen?" implies that the player already knows what they are, which doesn't fit well with his response. Maybe change it to a what is question like the other one?

> 				`	"Shipworm? How could this happen?"`

Author: ```suggestion
				`	"Shipworm? Never heard of it."`
```

			`	"Oh. Okay, carry on."`
				decline
			label curious
				`	He looks you up and down, perhaps wondering if he will get in trouble for telling you about another captain's ship, but then he shrugs and opens up.`

> 				`	"Shipworm infestation. Third one this week." He pushes his hood back and runs his hand through his greasy hair. "Honestly, I'm surprised the media hasn't been covering it yet. Maybe they're waiting for the first spontaneous depressurisation so they have something really juicy to talk about."

Author: It is indeed, good catch.

			`	"Shipworm infestation. Third one this week." He pushes his hood back and runs his hand through his greasy hair. "Honestly, I'm surprised the media hasn't been covering it yet. Maybe they're waiting for the first spontaneous depressurization so they have something really juicy to talk about."`
			choice
				`	"Shipworm? What is that?"`
				`	"You think it is a cover-up?"`
					goto coverup
			label whatisshipworm

> 			`	"They're tiny little creatures, about the size of a pinhead, and they get everywhere if you're not careful. They feed on a whole load of different metals, it has the science geeks blowing their tops. So far they haven't found an alloy that these guys can't digest."`

Reviewer 2: ```suggestion
			`	"They're tiny little creatures, about the size of a pinhead, and they get everywhere if you're not careful. They feed on a whole load of different metals, it has the science geeks blowing their tops. So far they haven't found an alloy that these guys can't digest.`
```

			label coverup

> 				`	"As a model Syndicate employee," he says sardonically, "I can tell you that the rumors that the shipworm is indigenous to this planet are untrue, and that the miners are categorically not hauling them out of the ground along with the ore. In fact, if you even use the term 'Delvan Shipworm' there is a good chance that you'll have Syndicate lawyers suing you for defamation."`

Author: ```suggestion
				`	"As a model Syndicate employee," he says sardonically, "I can tell you that the rumors that the shipworm is indigenous to this planet are untrue, and that the miners are categorically not hauling them out of the ground along with the ore. In fact, if you even use the term 'Delvan shipworm' there is a good chance that you'll have Syndicate lawyers suing you for defamation."`
```

> 			`	You thank the technician for his time and turn back toward the <ship>, wondering if maybe you should park her a bit further away.`

Reviewer: I feel like it would be better to refer to the ship as "it" outside of dialogue, since that's the standard way to pronoun ships these days.

(Having the technician use she/her in dialogue seems fine though)

```suggestion
			`	You thank the technician for his time and turn back toward the <ship>, wondering if maybe you should park it a bit further away.`
```

> 		log "Heard about some tiny creatures called 'Delvan Shipworms' which can eat through a ship's hull. Apparently infestations are getting more frequent."

Reviewer 2: ```suggestion
```

mission "Exotic Life: Ada"
	minor
	source Ada
	to offer
		random < 20
		has "Exotic Life: Delve: declined"
	on offer
		conversation
			`A loud commotion draws your attention as you make your way through the spaceport and you look up to see three large Lovelace Labs security guards surrounding an irate pilot.`
			`	"But I've got to unload my cargo!" the pilot is shouting right into the face of a guard who looks completely unfazed.`

> 			`	"Not here you're not," she says in a bored manner, as though she deals with this kind of situation twenty times a day. "Not until you get your ship decontaminated."`

Reviewer 2: Seems like a more natural response would be "you've not" to match what the other guy is saying. But her current response does sound more like she doesn't care much. Up to you whether to change it.

> 			`	"Not here you're not," she says in a bored manner, as though she deals with this kind of situation twenty times a day. "Not until you get your ship decontaminated."`

Author: ```suggestion
			`	"Not here you've not," she says in a bored manner, as though she deals with this kind of situation twenty times a day. "Not until you get your ship decontaminated."`
```

		`	The pilot looks suddenly sheepish, throwing his hands up in defeat.`
		`	"Alright, alright," he says, "I'll get the decon procedure done. Okay? You win."`
		`	"What. Are. They. Made of?"`
		`	"Stainless steel."`
		`	"Ouch," she says, but the bored expression is gone now and she seems to be enjoying herself. "Twelve tons of miniature figurines, and we are gonna have to scan every single one to make sure they're not contaminated. That could take a while."`
		`	With a scattering of chuckles and a few sympathetic groans the crowd starts to disperse, and the two other guards lead the pilot back towards his ship leaving the main guard tapping happily at her datapad. After a few moments she looks up, noticing you watching her.`
		choice
			`	"Delvan shipworms?"`
				goto shipworms
			`	"That seemed a bit harsh."`
			`	"That was fun."`
		`	She shrugs. "Just doing my job."`
			decline
				label shipworms
				`	"Uh-huh," she says. "Only we're now seeing them on ships that haven't been near Delve. Seem to be spreading."`

> 				decline

Reviewer 3: Hol up why are they showing as spaces??

> 				decline

Reviewer 3: ```suggestion

mission "Exotic Life: Delve"
	minor
	source Delve
	to offer
		random < 20
	on offer
		conversation
			`The mountain air tastes cool and fresh after breathing the constantly recycled air on the <ship> for so long, so you take a moment to get a good lungful and enjoy the view. Just off to your right, ground technicians wearing thin coveralls and breathing gear are spraying some kind of chemical on the hull of a parked Sunder. The fine mist reflects a bright rainbow which contrasts strikingly with the dull gray metal of the hull.`
			`	You wander over, careful not to get too close to the spray which you have to assume is toxic. One of the technicians climbs down a ladder attached to the hull as you approach, then turns to you, pulling their mask off to reveal a man in his early twenties, with dark skin and hair plastered with sweat to his forehead.`
			`	"This your ship?" he asks with a professionally neutral expression.`
			choice
				`	"No, I'm just curious. What are you doing to it?"`
					goto curious
				`	"Yes. How is the work coming along?"`
			`	The technician shakes his head morosely.`
			`	"It's confirmed. Shipworm. Bad case too, I'm surprised your hull was still holding atmo." He looks back at the Sunder, measuring her up with his eyes. "We should be finished with the decontamination in a couple hours, but then we'll need to go over her with sealant. Probably two coats. We'll be finished tomorrow afternoon, earliest."`
			choice
				`	"Shipworm? How could this happen?"`
					goto whatisshipworm
				`	"Oh. Okay, carry on."`
					decline
			label curious
			`	He looks you up and down, perhaps wondering if he will get in trouble for telling you about another captain's ship, but then he shrugs and opens up.`
			`	"Shipworm infestation. Third one this week." He pushes his hood back and runs his hand through his greasy hair. "Honestly, I'm surprised the media hasn't been covering it yet. Maybe they're waiting for the first spontaneous depressurization so they have something really juicy to talk about."`
			choice
				`	"Shipworm? What is that?"`
				`	"You think it is a cover-up?"`
					goto coverup
			label whatisshipworm
			`	"They're tiny little creatures, about the size of a pinhead, and they get everywhere if you're not careful. They feed on a whole load of different metals, it has the science geeks blowing their tops. So far they haven't found an alloy that these guys can't digest."`
			label coverup
				`	"As a model Syndicate employee," he says sardonically, "I can tell you that the rumours that the shipworm is indigenous to this planet are untrue, and that the miners are categorically not hauling them out of the ground along with the ore. In fact, if you even use the term 'Delvan Shipworm' there is a good chance that you'll have Syndicate lawyers suing you for defamation."`
			`	You thank the technician for his time and turn back toward the <ship> wondering if maybe you should park her a bit further away.`
			decline
		log "Heard about some tiny creatures called 'Delvan Shipworms' which can eat through a ship's hull. Apparently infestations are getting more frequent."

mission "Exotic Life: Ada"
	minor
	source Ada
	to offer
		random < 20
		has "Exotic Life: Delve: declined"
	on offer
		conversation
			`A loud commotion draws your attention as you make your way through the spaceport and you look up to see three large Lovelace Labs security guards surrounding an irate pilot.`
			`	"But I've got to unload my cargo!" the pilot is shouting right into the face of a guard who looks completely unfazed.`
			`	"Not here you're not," she says in a bored manner, as though she deals with this kind of situation twenty times a day. "Not until you get your ship decontaminated."`
			`	You are just one of a whole crowd of onlookers surrounding the group in a big circle now. The pilot seems to notice that he is the center of attention and decides to appeal to the crowd.`
			`	"You see what they're doing?" he yells. "This is... this is overreach! I'm being harassed!"`
			`	The guard is unimpressed, poking at a datapad as she talks.`
			`	"Rules are rules and you gotta follow them just like everyone else." She looks up, eyes narrowed. "It says here your cargo is miniature figurines. What exactly are they made out of?"`
			`	The pilot looks suddenly sheepish, throwing his hands up in defeat.`
			`	"Alright, alright," he says, "I'll get the decon procedure done. Okay? You win."`
			`	"What. Are. They. Made of?"`
			`	"Stainless steel."`
			`	"Ouch," she says, but the bored expression is gone now and she seems to be enjoying herself. "Twelve tons of miniature figurines, and we are gonna have to scan every single one to make sure they're not contaminated. That could take a while."`
			`	With a scattering of chuckles and a few sympathetic groans the crowd starts to disperse, and the two other guards lead the pilot back towards his ship leaving the main guard tapping happily at her datapad. After a few moments she looks up, noticing you watching her.`
			choice
				`	"Delvan shipworms?"`
					goto shipworms
				`	"That seemed a bit harsh."`
				`	"That was fun."`
			`	She shrugs. "Just doing my job."`
				decline
			label shipworms
			`	"Uh-huh," she says. "Only we're now seeing them on ships that haven't been near Delve. Seem to be spreading."`
				decline
```

				payment -294
			`	You fish out the money and hand it to him. He rubs it, then explodes in rage. "Get out!" he shouts, throwing the credit chips at your face. You make a run for it, and by the time you get to your ship, you realize that you left 294 credits with the old man in your hurry to escape.`
				decline

mission "Exotic Life: Delve"
	minor
	source Delve
	to offer
		random < 20
	on offer
		log `Heard about some tiny creatures called "Delvan shipworms" which can eat through a ship's hull. Apparently infestations are getting more frequent.`
		conversation
			`The mountain air tastes cool and fresh after breathing the constantly recycled air on the <ship> for so long, so you take a moment to get a good lungful and enjoy the view. Just off to your right, ground technicians wearing thin coveralls and breathing gear are spraying some kind of chemical on the hull of a parked Sunder. The fine mist reflects a bright rainbow which contrasts strikingly with the dull gray metal of the hull.`

> 			`	You wander over, careful not to get too close to the spray, which you assume is toxic. One of the technicians climbs down a ladder attached to the hull as you approach, then turns to you, pulling their mask off to reveal a dark-skinned man in his early twenties, his hair plastered to his forehead by copious amounts of sweat.`

Reviewer: Hmm... does it make more sense to have this as "pulling his mask off" so it's consistent? Could go either way.

Reviewer 2: I agree, we should use his here

Author: ```suggestion
			`	You wander over, careful not to get too close to the spray, which you assume is toxic. One of the technicians climbs down a ladder attached to the hull as you approach, then turns to you, pulling his mask off to reveal a dark-skinned man in his early twenties, his hair plastered to his forehead by copious amounts of sweat.`
```
Makes sense to me.

			choice
				`	"Shipworm? Never heard of it."`
					goto whatisshipworm
				`	"Oh. Okay, carry on."`
					decline
			label curious
			`	He looks you up and down, perhaps wondering if he will get in trouble for telling you about another captain's ship, but then he shrugs and opens up.`
			`	"Shipworm infestation. Third one this week." He pushes his hood back and runs his hand through his greasy hair. "Honestly, I'm surprised the media hasn't been covering it yet. Maybe they're waiting for the first spontaneous depressurization so they have something really juicy to talk about."`
			choice
				`	"Shipworm? What is that?"`
				`	"You think it's a cover-up?"`
					goto coverup
			label whatisshipworm
			`	"They're tiny little creatures, about the size of a pinhead, and they get everywhere if you're not careful. They feed on a whole load of different metals, it has the science geeks blowing their tops. So far they haven't found an alloy that these guys can't digest."`
			label coverup

> 				`	"As a model Syndicate employee," he says sardonically, "I can tell you that the rumors that the shipworm is indigenous to this planet are untrue, and that the miners are categorically not hauling them out of the ground along with the ore. In fact, if you even use the term "Delvan shipworm" there is a good chance that you'll have Syndicate lawyers suing you for defamation."`

Reviewer 2: This one might be on me.
Since we've got double quotes around what he's saying already, it should be single quotes here.
Theoretically this will also fix the check.
```suggestion
				`	"As a model Syndicate employee," he says sardonically, "I can tell you that the rumors that the shipworm is indigenous to this planet are untrue, and that the miners are categorically not hauling them out of the ground along with the ore. In fact, if you even use the term 'Delvan shipworm' there is a good chance that you'll have Syndicate lawyers suing you for defamation."`
```

Author: No, it's just me over-generalising. Thanks for catching it.

			`	"Not here you've not," she says in a bored manner, as though she deals with this kind of situation twenty times a day. "Not until you get your ship decontaminated."`
			`	You are now just one of a whole crowd of onlookers surrounding the group in a big circle. The pilot seems to notice that he is the center of attention and decides to appeal to the crowd.`
			`	"You see what they're doing?" he yells. "This is... this is overreach! I'm being harassed!"`
			`	The guard is unimpressed, poking at a datapad as she talks.`
			`	"Rules are rules and you gotta follow them just like everyone else." She looks up, eyes narrowed. "It says here your cargo is miniature figurines. What exactly are they made out of?"`
			`	The pilot looks suddenly sheepish, throwing his hands up in defeat.`
			`	"Alright, alright," he says. "I'll get the decon procedure done. Okay? You win."`
			`	"What. Are. They. Made of?"`
			`	"Stainless steel."`
			`	"Ouch," she says, but the bored expression is gone now and she seems to be enjoying herself. "Twelve tons of miniature figurines, and we are gonna have to scan every single one to make sure they're not contaminated. That could take a while."`

> 			`	With a scattering of chuckles and a few sympathetic groans the crowd starts to disperse, and the two other guards lead the pilot back towards his ship leaving the main guard tapping happily at her datapad. After a few moments she looks up, noticing you watching her.`

Reviewer 2: ```suggestion
			`	With a scattering of chuckles and a few sympathetic groans the crowd starts to disperse, and the two other guards lead the pilot back towards his ship, leaving the main guard tapping happily at her datapad. After a few moments she looks up, noticing you watching her.`
```
